operate/pom.xml

```diff
@@ -170,20 +170,13 @@
     </profile>
 
     <profile>
-      <!--
-        We have to skip spotbugs analysis until Operate team have fixed several findings
-        See https://github.com/camunda/zeebe/issues/17331
-      -->
       <id>spotbugs</id>
       <build>
         <plugins>
           <plugin>
             <groupId>com.github.spotbugs</groupId>
             <artifactId>spotbugs-maven-plugin</artifactId>
             <version>${plugin.version.spotbugs}</version>
-            <configuration>
```

Comment:
I see that `spotbugs-maven-plugin` is also declared in parent POM  and Zeebe is using `spotbugs` plugin to trigger the check (e.g. [here](https://github.com/camunda/zeebe/blob/57c33486e3cf0fcda20b04ccadd806583ad258f0/.github/workflows/zeebe-ci.yml#L504)). You decided not to go that way and rather have it always running and explicitly disabling by choice?